tvloader/parser2v2/parse_creation_info.go

```diff
@@ -12,7 +12,7 @@ import (
 func (parser *tvParser2_2) parsePairFromCreationInfo2_2(tag string, value string) error {
 	// fail if not in Creation Info parser state
 	if parser.st != psCreationInfo2_2 {
-		return fmt.Errorf("Got invalid state %v in parsePairFromCreationInfo2_2", parser.st)
+		return fmt.Errorf("got invalid state %v in parsePairFromCreationInfo2_2", parser.st)
```

Comment:
I'm not going to reproduce all of them, but I think all of my comments for the 2_1 code above also apply to the 2_2 code here, and should be handled similarly.